tests/composer.json

```diff
@@ -115,6 +116,17 @@
                     "create-lockfile": false
                 }
             },
+            "opentelemetry1": {
+                "require": {
+                    "open-telemetry/sdk": "^1.0.0",
```

Comment:
Is there a way to ask for latests? It would be great as we would have tests by default against the latest version

Reply:
Changed it to use `@stable` instead